### ISSUE TITLE
Read-only props

### ISSUE THREAD

**Issue by isBatak** [infinum/datx#153]
Submitting a:

* [ ] Bug report
* [x] Feature suggestion
* [ ] Support request

Props could be marked read-only, so they are never sent to the server on save.
```
@prop({ persist: false }) createdAt: string;
@prop({ persist: false }) updatedAt: string;
```

**Comment by DarkoKukovec** [comment]
I agree this could be useful. The problem right now is that `@prop` is very tightly coupled with datx and therefore it's tricky to add some jsonapi specific things. In datx 2, the decorator will be more flexible, so it should be simpler to  add this feature. Until then, you can override the `toJsonApi` method on the model and remove the properties from the payload.

**Comment by DarkoKukovec** [comment]
Until this is implemented, #154 can be used as a workaround:
```typescript
@Attribute({ serialize: () => undefined })
```
